mkimage.sh

```diff
@@ -100,6 +100,12 @@ ENV DEBIAN_FRONTEND noninteractive
 LABEL io.resin.architecture="rpi" \
 	  io.resin.qemu.version="$RESIN_QEMU_VERSION" \\
 	  io.resin.device-type="raspberrypi"
+RUN apt-get update && apt-get install -y --no-install-recommends wget \
+	&& wget http://archive.raspberrypi.org/debian/raspberrypi.gpg.key \
+	&& apt-key add raspberrypi.gpg.key \
+	&& rm -rf raspberrypi.gpg.key \
+	&& echo "deb http://archive.raspberrypi.org/debian/ jessie main" >> /etc/apt/sources.list \
```

Comment:
@petrosagg, `apt-get` handles this case pretty well.
```
Several instances of the same version of a package may be available when the sources.list(5) file contains references to more than one source. In this case apt-get(8) downloads the instance listed earliest in the sources.list(5) file. The APT preferences file does not affect the choice of instance, only the choice of version.
```
ref: http://linux.die.net/man/5/apt_preferences
So I don't think this will be a problem